### PR TITLE
feat(crashlytics): add helper methods for log and setCustomKey

### PR DESCRIPTION
**Pull request by acailly**
### Description

This add additionnal helpers in ReactNativeFirebaseCrashlyticsNativeHelper to call FirebaseCrashlytics native functions

We have a native util class `CrashlyticsUtils.java` that is used to  centralize crashlytics interaction in native side
We would like to ensure this class uses the same crashlytics dependency than the RNFirebase module, via these additionnal helpers

### Checklist

- I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [X] Yes
- My change supports the following platforms;
  - [X] `Android`
  - [ ] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `packages/\*\*/e2e`
  - [ ] `jest` tests added or updated in `packages/\*\*/__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [X] No

### Test Plan

---

Think `react-native-firebase` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

**Review by mikehardy — DISMISSED** [review]
Hey there! Sure, I had to go back in git history for a moment to make sure we were going in a healthy direction here (having a native helper at all...) but now I remember the motivation well and the use case it was addressing. This still seems like the best way to handle consolidated crash logging in a mixed react-native / native app so adding these methods also seems fine.

I'll approve the CI and merge this when it's all green

Thanks!

**Comment by mikehardy** [comment]
@acailly oh - and you *must* sign the CLA prior to merge, can't merge it without even though the change is tiny

**Comment by acailly** [comment]
@mikehardy thank you for the quick review 😃 
I used the wrong email so I changed the commit email and signed the CLA, this should be ok

**Review by mikehardy — APPROVED** [review]

**Pull request merged**
